packages/api/cms-api/src/user-permissions/user-permissions.service.ts

```diff
@@ -195,4 +198,23 @@ export class UserPermissionsService {
                 return p;
             });
     }
+
+    async hasEqualOrMorePermissions(user: User | CurrentUser, targetUser: User): Promise<boolean> {
+        const permissions = "permissions" in user ? user.permissions : await this.getPermissionsAndContentScopes(user);
+        const targetPermissions = await this.getPermissionsAndContentScopes(targetUser);
+        for (const permission of targetPermissions) {
+            const currentUserPermission = permissions.find((p) => p.permission === permission.permission);
+            if (!currentUserPermission) {
+                console.log("Permission not found", permission.permission);
+                return false;
+            }
+            for (const contentScope of permission.contentScopes) {
+                if (!currentUserPermission.contentScopes.find((cs) => isEqual(cs, contentScope))) {
+                    console.log("ContentScope not found", contentScope);
```

Comment:
These console logs seem like they are for debugging purposes and should be removed